### PR TITLE
feat: pips blog

### PR DESCRIPTION
**Pull request by kehiy**

### CODE REVIEW

**Comment by kehiy** [comment]
@b00f can you merge it?

File: website/_posts/2023-09-04-what-is-a-pip.md [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
+Many projects in the tech world utilize similar conventions like BIP, EIP, PEP, and so on.
 
-These names are just short sentences for:
+These acronyms stand for:
 
-* Bitcoin improvement proposal
-* Ethereum improvement proposal
-* Python enhancement proposal
-* Nostr improvement possibilities
+* Bitcoin Improvement Proposal
```
Comment (b00f) [inline website/_posts/2023-09-04-what-is-a-pip.md:21]: Not bad add links here

File: website/_posts/2023-09-04-what-is-a-pip.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+These acronyms stand for:
+
+* Bitcoin Improvement Proposal
+* Ethereum Improvement Proposal
+* Python Enhancement Proposal
+* Nostr Improvement Possibilities
+
+These are specifications that guide projects in their improvement efforts.
+hey are often numbered, such as PIP-1, PIP-2, etc.
```
Comment (kehiy) [inline website/_posts/2023-09-04-what-is-a-pip.md:27]:
```suggestion
They are often numbered, such as PIP-1, PIP-2, etc.
```

File: website/_posts/2023-09-04-what-is-a-pip.md [hunk 3]
```diff
@@ -1,36 +1,38 @@
...
 
-These are some specification that helps to project to improve, They have numbers also like PIP-1, PIP-2, and …
+These specifications help steer the direction of a project.
+They are usually assigned unique numbers like PIP-1, PIP-2, and so on.
 
-Everyone can propose one of them for each project.
+Anyone can propose these for any project.
 
-Now we have Pactus improvement proposals [here](https://github.com/pactus-project/PIPs)
+Currently, Pactus Improvement Proposals can be found [here](https://github.com/pactus-project/PIPs).
```
Comment (b00f) [inline website/_posts/2023-09-04-what-is-a-pip.md:36]: https://pips.pactus.org/

File: website/_posts/2023-09-04-what-is-a-pip.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+These acronyms stand for:
+
+* [Bitcoin Improvement Proposal](https://github.com/bitcoin/bips)
+* [Ethereum Improvement Proposal](https://github.com/ethereum/EIPs)
+* [Python Enhancement Proposal](https://github.com/python/peps)
+* [Nostr Improvement Possibilities](https://github.com/nostr-protocol/nips)
+
+These are specifications that guide projects in their improvement efforts.
+hey are often numbered, such as PIP-1, PIP-2, etc.
```
Comment (kehiy) [inline website/_posts/2023-09-04-what-is-a-pip.md:27]:
```suggestion
They are often numbered, such as PIP-1, PIP-2, etc.
```

**Review by b00f — APPROVED** [review]

**Pull request merged**